From 092eb1db5a0cf778fe4d02051916d62e6fa2b897 Mon Sep 17 00:00:00 2001
From: Mike Nerone <mike@nerone.org>
Date: Tue, 18 Feb 2014 01:18:05 -0600
Subject: [PATCH] Update all the Minecraft things...

Tweaks for servers:

A. 1.7 changed location of log lock file.
B. That file now remains when stopped, so ewaitfile doesn't work for
waiting for the PID. Replaced with a fuser-check loop.
C. minecraft-common version changed to 1.7 to reflect the restructuring
that happened in the 1.7 series (e.g. the above) with which it is
compatible. Because of such changes, individual server versions depend
on having the corresponding minecraft-common; this versioning scheme
provides a convenient way to ensure that via ebuild dependencies.
---
 games-action/minecraft/Manifest                        | 18 ++----------------
 ...inecraft-1.2.1-r4.ebuild => minecraft-1.3.9.ebuild} |  0
 games-server/bukkit/Manifest                           |  4 ++--
 .../bukkit/{bukkit-1679.ebuild => bukkit-1903.ebuild}  |  2 +-
 games-server/craftbukkit/Manifest                      |  6 +++---
 ...craftbukkit-2624.ebuild => craftbukkit-3020.ebuild} |  6 +++---
 games-server/minecraft-common/Manifest                 |  4 ++--
 games-server/minecraft-common/files/init.sh            | 11 +++++++++--
 ...mon-20130203.ebuild => minecraft-common-1.7.ebuild} |  0
 games-server/minecraft-server/Manifest                 |  4 ++--
 ...rver-1.4.7.ebuild => minecraft-server-1.7.4.ebuild} |  4 ++--
 11 files changed, 26 insertions(+), 33 deletions(-)
 rename games-action/minecraft/{minecraft-1.2.1-r4.ebuild => minecraft-1.3.9.ebuild} (100%)
 rename games-server/bukkit/{bukkit-1679.ebuild => bukkit-1903.ebuild} (98%)
 rename games-server/craftbukkit/{craftbukkit-2624.ebuild => craftbukkit-3020.ebuild} (97%)
 rename games-server/minecraft-common/{minecraft-common-20130203.ebuild => minecraft-common-1.7.ebuild} (100%)
 rename games-server/minecraft-server/{minecraft-server-1.4.7.ebuild => minecraft-server-1.7.4.ebuild} (94%)

diff --git a/games-action/minecraft/Manifest b/games-action/minecraft/Manifest
index efcf3e6..8556f9d 100644
--- a/games-action/minecraft/Manifest
+++ b/games-action/minecraft/Manifest
@@ -1,20 +1,6 @@
------BEGIN PGP SIGNED MESSAGE-----
-Hash: SHA256
-
 AUX minecraft-gentoo 4894 SHA256 938d6cf28bb9ff529db5225bda7edba1ef1f975a11b80b2f64f4721ceef5ecb4 SHA512 821c6301636256c354b36965cdbb7ae5f2948ca37325c22cefb81766c95ce8f674de15c94267451b6deaa8af409641b77067a6e347009a3535254d89836c327f WHIRLPOOL 75d4f1d355bdbcee217ce30b8e1cc5b22e68613e35f307a758d55ebd5ce89167163c4fb40cae20b3bb671f963fec56ea101fb2a041a3d8cbc60bc3de384cb130
 AUX minecraft.png 20345 SHA256 5ee86aeb4185637ea901f80e698e64d07dd138448fc4ff00d96d5992792a806b SHA512 ce91da9e858dd4ab63cd5643304d59ad7169788638cd8a59edce14f62cf1723896e9e8280cc0645930f2b1ba2e1202b4755280c2d546abba263a83d32e3770e0 WHIRLPOOL b33a8599e7ebffdb7a92d4edb59a20492658c34f76a301b723dcdaba16e120c3c04800b174be3f652687cb349742e5cacbff3cbe30672e8da1dc5a10b23534d3
-DIST minecraft-1.2.1.jar 280212 SHA256 e8d90b2637b141b556670d8460d317aadb6e4f2e644e519995c428800b25f212 SHA512 64201f5d9522c43bded397e39b29eedeed5af224a064e13eede6f5d276d7b61702c5daae998c1fff8b34c6660f8f56a40bf0f43f4024119023c77672d6319659 WHIRLPOOL 3ecf370c4c99b6ae7b33efba5ca831c360b8c72ece5f059bbeb7c5e8e7686d6562338904522e907b5b1acaca17fe99b97e985ea770f8f4f20d6040d817e4f312
-EBUILD minecraft-1.2.1-r4.ebuild 2782 SHA256 4a5500aec99b006c3194a8a4ec60220fbeaa6d0e6e89d9bbec175876b847f11c SHA512 6bc2940dae5f4967545fb116dfd3a7e36295e3b8cc7ca4a9fa0956ac8f46d7a9b26e301ea8d38a9de6811bf57d04d57e12dd17a4d9b13c00622919c1ea74e778 WHIRLPOOL 0e5d51fb971a8c723eda8f9f5cb798e34ed02c4f74bd9f7009d7cb38f000742718482c839a880f81ed612d4289931d780123b924a365ebb8351e95337c6654a7
+DIST minecraft-1.3.9.jar 280212 SHA256 e8d90b2637b141b556670d8460d317aadb6e4f2e644e519995c428800b25f212 SHA512 64201f5d9522c43bded397e39b29eedeed5af224a064e13eede6f5d276d7b61702c5daae998c1fff8b34c6660f8f56a40bf0f43f4024119023c77672d6319659 WHIRLPOOL 3ecf370c4c99b6ae7b33efba5ca831c360b8c72ece5f059bbeb7c5e8e7686d6562338904522e907b5b1acaca17fe99b97e985ea770f8f4f20d6040d817e4f312
+EBUILD minecraft-1.3.9.ebuild 2782 SHA256 4a5500aec99b006c3194a8a4ec60220fbeaa6d0e6e89d9bbec175876b847f11c SHA512 6bc2940dae5f4967545fb116dfd3a7e36295e3b8cc7ca4a9fa0956ac8f46d7a9b26e301ea8d38a9de6811bf57d04d57e12dd17a4d9b13c00622919c1ea74e778 WHIRLPOOL 0e5d51fb971a8c723eda8f9f5cb798e34ed02c4f74bd9f7009d7cb38f000742718482c839a880f81ed612d4289931d780123b924a365ebb8351e95337c6654a7
 MISC ChangeLog 6433 SHA256 5a7b642a2701a4d9f42887a406bfaf6b847e0a15bc943b736a070cf23d3175fe SHA512 30cae5fd13c2395fd8f60170dac6a4f3b6bdbeca230dafa0f7f0246d9cfbad368406ebfc1f1e7b319ea3ea0bcf9fb1ce35b135033b4116df3e78d526c23e149a WHIRLPOOL dd4bd006cb685683fe12bbb1d7bcdbfa2a7a45f603c64e014fc239638e4a61c27e6b9b9bd35ad2207f939fe1f400e0c60a24f6eb9aaa00737be4969b9a02bd94
 MISC metadata.xml 525 SHA256 e9e868a41f0ca909166b4977b6188316196ef4016f0035f2ec18b3a2696ec918 SHA512 7e854e616459ccb206389144b5caf7f613632ee27fa718664e649c956d71a5652eff6aa88f1d85a5c16c76e83349b4ebeacacddb44c37320196252b869ff7d62 WHIRLPOOL 3a00529b7b3dad61a7158e86a4de5e154420772eeedf55c0fccc39da6b1b35317e75a2b0d0856d907896e60dfd134f200d8cff2d896c45701a1852cf29d2cc88
------BEGIN PGP SIGNATURE-----
-Version: GnuPG v2.0.20 (GNU/Linux)
-
-iQEcBAEBCAAGBQJSLj9yAAoJEGcL3XYFj+mJ0zIH/3weHvHp7/m5PGfCZraxCbJo
-GB0V0T0jAZMEi83HRFNmlyFaFVz7oyOmxEVKMf9RHXWmfYomLV6Vy+xHItl35Jjm
-lgy1vqXEo3ruSP/kF8gWqFX/tcZw59Cbt6pijeKdb3hUYh7KYhxrVrQR/GFqzsPQ
-KTpuFGcmOx/+T2WmF60S9VAMrO0BA3ux1uGQAdowGVyqVEc6NdPYdmrmcTueBLUD
-faOE0lFrhapDCQ5uFr0mEc7wCbeBjXuGXnV5ajbWyGchK22KaomeG5WRJPxY3hBM
-bIUGSIovpcwujQsEB/KaPbM0bL5ucIYca9bKsqHT6YttdadgL4H6E6BLnyZDFiQ=
-=CT47
------END PGP SIGNATURE-----
diff --git a/games-action/minecraft/minecraft-1.2.1-r4.ebuild b/games-action/minecraft/minecraft-1.3.9.ebuild
similarity index 100%
rename from games-action/minecraft/minecraft-1.2.1-r4.ebuild
rename to games-action/minecraft/minecraft-1.3.9.ebuild
diff --git a/games-server/bukkit/Manifest b/games-server/bukkit/Manifest
index eab4910..b5a8b68 100644
--- a/games-server/bukkit/Manifest
+++ b/games-server/bukkit/Manifest
@@ -1,4 +1,4 @@
-DIST bukkit-1679.tar.gz 302805 SHA256 319fab38baa44ff95c1bf62a747a482890bd008bb3a045670588f02115b56780 SHA512 069116fccb4d652ba84446cdf47826b35cbd58c6f771d62a9edf196647cf51f4547a41d65b88ce8fa8855e5004cfa1864b78586cca129f047855a1aa9ca42259 WHIRLPOOL aa80f959b7a67149b15ee41c5fd22a20712c6f6dacec242ffb8583d89f803d11a24178f661c9016a577171d21b2aad9f9a6ba584b9303e9c4356023e72f7803b
-EBUILD bukkit-1679.ebuild 1548 SHA256 afab310da9155022b1410e48d56cb400b016d361491a9b6126e27542a228e8e5 SHA512 4acb62dc4c3d75333a00aa682aa8a7c7f0f5b08a70282e8f2e54f8c54662749aa9ac2730063b998f22f0c2a4eb59af3ad562abdfad5c8fce617dff4d3a0cc13e WHIRLPOOL 37d5fb313dc538fb81e35ef4cc20cf98e288139d40bc49aee20dbab0bce64b677d4802d84e83c2805dea7d6cbe57fbcc4e6390159cd7628c4f1d455bf6155030
+DIST bukkit-1903.tar.gz 371349 SHA256 980aced3f63efbd8897716d25d31f1c82e6cd327475289aec88f8b1538e623d1 SHA512 4009d27040e80487b70c25d54531a208addf78f29d0d5975690e94ccd635c0bf2bca5cc6afdd301447fd36aafe7aff9282cd3b068092f4efcbe61e62bfb1272e WHIRLPOOL dd589dbab5f945ca9aed403c8f8b277af0d3892959450d58ff7108fd814516990f81b3e35d6c152bfcbcfc47b4844917409bcd1b0affae2632da4a8ee3e77d85
+EBUILD bukkit-1903.ebuild 1548 SHA256 f1c2ca35769db84906080a6539206d09f9a48fe2e24b7979609e579d1e2c0d08 SHA512 c16907928ff57b81d949a16244ba1046fa1f7a03afb28426e1e3b5d93ac3704dbeccfce46ed19a1742e03ddc21d1090cf6a36dd15171a0d047670e4668a395f3 WHIRLPOOL 64af5ac6f49232dd24ed916d447fd1977f01324952a530a7d1a95cabb68c2c0b9672d9a88110bb4b9235465c87b9d982255478fca958c9c4b2cbfa891a4a18f8
 MISC ChangeLog 2243 SHA256 e20d84f4b395ba1da80f500f99f8cd975ead7a5e715150f327f3534edbf299b1 SHA512 154bdc93194cd156802d438fea4400c9072c00f357808081abcc7671a59e8e19a9e00866eefc481c73992fb0e2ad888e21d20a4955c6aafed61476da9e0345b2 WHIRLPOOL 7c362dde3cbe60762a313b9f26acb7aa5c2805cf93e21dc08b7e3558443a9f76da1eaf6b63205c71bfa58aec598faae591d49736c38a2a838bc2d181cf3e5a0e
 MISC metadata.xml 491 SHA256 c3938aa50fbab6958ff73cab178e8fe29c2db1f6e2f552d116ae31b7b84e1b89 SHA512 ad6fad3a7aabef6ac8a08fe11d1af600a3c17a853077ff15fb64377036f1d3ef05cbe86d72195bfc63427169a1e49c88eb0432df45f8f973918cf504dc00b1aa WHIRLPOOL 542c2a35263c0fbd3162b5a1a50f173808528c07a614f9850e93976824b494880c7bca1996b05c610711d2cbb848d36a5b5a129182f4e5fe65f29ff558b0e6c1
diff --git a/games-server/bukkit/bukkit-1679.ebuild b/games-server/bukkit/bukkit-1903.ebuild
similarity index 98%
rename from games-server/bukkit/bukkit-1679.ebuild
rename to games-server/bukkit/bukkit-1903.ebuild
index 54adfb1..f184221 100644
--- a/games-server/bukkit/bukkit-1679.ebuild
+++ b/games-server/bukkit/bukkit-1903.ebuild
@@ -3,7 +3,7 @@
 # $Header: $
 
 EAPI=4
-MY_PV="1.4.7-R1.0"
+MY_PV="1.7.2-R0.3"
 JAVA_PKG_IUSE="doc source"
 
 inherit games vcs-snapshot java-pkg-2 java-pkg-simple
diff --git a/games-server/craftbukkit/Manifest b/games-server/craftbukkit/Manifest
index fffe559..e7c4485 100644
--- a/games-server/craftbukkit/Manifest
+++ b/games-server/craftbukkit/Manifest
@@ -1,6 +1,6 @@
 AUX directory.sh 869 SHA256 f0345da259f2de1e81c5fac06d0a779040bb99e9ce06089fec6f362296598b5d SHA512 12e906af6d804c26ba9a253a552786a3346a2524fee36ebb63101cca18fef8124460d98f6612aeb080c7b542e39f4bb6ec49d48e5279ec4c88ad3c54de0c1873 WHIRLPOOL 413c60fb6fc876f3ef3165c364b1edf2d096266e798b8b76fa39e2719dbea4697dce3655090c06a6e806d6a927596edba76c5645cf73906f3592ec2ec2c996c0
-DIST craftbukkit-2624.tar.gz 484480 SHA256 c9323201e9a9700f16341c56b5196d136cb9096a7b476a23ba283f59264db0d2 SHA512 8f0a19f6913a88e365c7fb6a0515577a63c04d0d8cf105421ea70fd8cb2bc2a62ba04fc4112cd09f46534f510a8295c99a4fa5876a16cc6bdd58c0fd783a06da WHIRLPOOL fde51dc3c1d6170b316751ab4cfd62d6fa72cf7ad89e81b7aa687f1341e8912d5906fcf3887ebf1f7d75db044e7ff3db0c8f2c8b0071a557d9606f0699be2581
-DIST minecraft-server-unobfuscated-1.4.7.jar 2331365 SHA256 942bb1700fdd09aada0b1389476ff133049d91c92a5de9b82baf0043e00791e4 SHA512 72ecf4396bfc1ae6d25e2a129d23d59a659d516866a8e3fefbcbc5e566968cd1c995606b0608a31ac5efa31ed8594b071e9f6bde5719eb1f3ac1a22538ca88b9 WHIRLPOOL 63d262e70e9e2f6774b4c1adf74f9849b443a302183884d2707a002b6aefadd524909ae59697824cbecfe7e0ffd2cd5130c795e1a9877aeff77c42cb26b7674a
-EBUILD craftbukkit-2624.ebuild 4169 SHA256 d1b9cc87739a1290c15b929f328fea59c8e6579accebbf20f9655696d734c8e2 SHA512 8362d30e9e2fd2675f0918d12f800abd03bfb756905ab07a0ab38f89e42088749c4b7e84681cf3cb016df73b6d32a1d5cfe655190ba4488866476fb2e6c693b3 WHIRLPOOL dd56dce1b72e1bad37705d4c8a85eadd0c996e6a87a76a5aa5de6290c3efaecd7d00a0d0bd65f8e9eb015fd69ba4ddf7041b9a0c6dd018b6c880fd226943b2dd
+DIST craftbukkit-3020.tar.gz 587351 SHA256 429f8d7e587faa8c25fe6cfae8a272b754b8ca83440e8dadd5ba6b55995cf425 SHA512 2bf848cc8ff5b15ad79f0cc0f166f6cac81cf21a0e0bf1aaea5f1ce22ff8d6fad971f0dbe667dc39a428ca62e30f30829bdd71ee1fd0639c5f2e9730b5436031 WHIRLPOOL 1efb145233667e8e36908d666741c32bba19cd2fa30a63cd78e0fa66a1425b6ece5c53af72037161c6eda6cb08185ce78205fbcebc724ac98979a6253142f5db
+DIST minecraft-server-unobfuscated-1.7.2.jar 9872003 SHA256 1356e81b12e49b8151543b11ca39a610f314bb47cf029a49801ac149d679d898 SHA512 2d21427097a6d9a379f4edaaf326bd817b0c3ad088370e2ba30b5aa375bc014ab898c3b66c2d9167c3371b7d6e8e0e7d04e57d696b95be9e887fe65a7ce6f39e WHIRLPOOL e7dcf2ca1ff1e51767b25106ec220f88131503aa40ccf9f2b6aa80bb4bb0c9a3c2eb408712435cd30e82d05f7854aabef69c328625d6e2c830b0197b485491c6
+EBUILD craftbukkit-3020.ebuild 4174 SHA256 adf8921385709a143761730f426bf7a77415411dd57f14403f84cd182dd3bc77 SHA512 ccad15d59477764ee3e9b179538dab974381b55a507e2b2a8718fbdb8b04f9f286fca9791454b77093588ca27e4692a656b2702932c04af6d26c5ceffc8ef72d WHIRLPOOL 98e496e88ce05017ea81944764c31aed47f3534c2c21c5dc1c0c38e49b0468fa9927e502cab30d5f3c689236734dbb392ed627e2d148ea4acca83eb3dddb8e6e
 MISC ChangeLog 2505 SHA256 87150ae5faf3a616c19363e79d19171a5a3ffbb221a8860887941a938fdbbb15 SHA512 5c3d4804ea8c44a1dbff6fd1b0276670f4032d6481d6265bfcf4861272ef0f61c4f64ba39bae8993ffc202d6e55b91bd1a2d3149feb4221fd8679506051479fb WHIRLPOOL eea31b673286684e99729ba284c764d3b6b3265bcec6dd9e2d2eb72b6a34b3d40f957b60ddb15686aa94c2803c14ef967b2f4ef629757a3039550afdba565589
 MISC metadata.xml 436 SHA256 5d12675555bdd758a7ca0419c621309004e109b001d2e2e9ee0ddfdc8dfd555f SHA512 7a7d8e66b4c1e05a8d5450da97a98f5b90a7b62d2ae4fe5fd93a7155324e94f2d4cf239f3bc0ae2759573c910ef1d65c49ff47785aaf6d3ae52dd3479d319f5e WHIRLPOOL a8f91ff90590912b76f0aa34aa4df15e3d5e966051ca213bb9c903281ea0a291b8c3d5c2febe970b66232e898c922ba23891b9b4613ed61d2d67080d8b240a79
diff --git a/games-server/craftbukkit/craftbukkit-2624.ebuild b/games-server/craftbukkit/craftbukkit-3020.ebuild
similarity index 97%
rename from games-server/craftbukkit/craftbukkit-2624.ebuild
rename to games-server/craftbukkit/craftbukkit-3020.ebuild
index 1919102..1f2bba5 100644
--- a/games-server/craftbukkit/craftbukkit-2624.ebuild
+++ b/games-server/craftbukkit/craftbukkit-3020.ebuild
@@ -3,7 +3,7 @@
 # $Header: $
 
 EAPI=4
-MY_PV="1.4.7-R1.0"
+MY_PV="1.7.2-R0.3"
 MC_PV="${MY_PV%-*}"
 MC_PN="minecraft-server-unobfuscated"
 MC_JAR="${MC_PN}-${MC_PV}.jar"
@@ -28,7 +28,7 @@ CDEPEND="dev-java/commons-lang:2.1
 	dev-java/jline:2
 	dev-java/jopt-simple:0
 	>=dev-java/snakeyaml-1.9:0
-	~games-server/bukkit-1679:0"
+	~games-server/bukkit-1903:0"
 
 DEPEND="${CDEPEND}
 	>=virtual/jdk-1.6"
@@ -37,7 +37,7 @@ DEPEND="${CDEPEND}
 
 RDEPEND="${CDEPEND}
 	>=virtual/jre-1.6
-	games-server/minecraft-common"
+	~games-server/minecraft-common-1.7"
 
 S="${WORKDIR}/${P}"
 
diff --git a/games-server/minecraft-common/Manifest b/games-server/minecraft-common/Manifest
index 022a776..5eddcb1 100644
--- a/games-server/minecraft-common/Manifest
+++ b/games-server/minecraft-common/Manifest
@@ -1,6 +1,6 @@
 AUX console.sh 370 SHA256 e14e7f85104c8662f6938b9f100ccf23abda4b6828484a7fc1d39be3c5af4289 SHA512 b96ea73bd93d603ef259d3c21453ea691ed3b0429af888e2c2978e5195599a82fb4d76fe3b6ab03adb9bc101237beeb78a0ff114c110f1bccdc3ba63a43224a9 WHIRLPOOL da3856731710e896cab1127b0d47291bccb24d68f88ad203116b2af4ffac9a6dd9d053668b4d5d9f9cc9c782fa5dff40457e33f8970ba0fbae512804e752edbb
-AUX init.sh 1374 SHA256 a2ca345803dabac3caf13af4f74c600f39e9bcbd510e03a44a681e758cd3be04 SHA512 8da4026761f431a16312c1a7a72195503f72b76678fecbd414d65f4d3ff588644a983b4a91d6946197d368b34d1073c204b63e329beb9abf17463e3ea49ba4c7 WHIRLPOOL 43e5f0393e749107d02e61d0f88575b10979884fdf5983e67e6607145b0b4a1269a7f672dcaa785529bfc6963b9cd90a3190e690d6fd0490fa10f5fdede0d568
+AUX init.sh 1472 SHA256 6d67022477d64d9461e4c18dc98d1bd313be3778b21935bfe9cb50de7bb4aaf6 SHA512 52111118e4d98055646f34c71c9a43c9a63cdda5d016062fe54c728e81129f5596c86b7b69311722f9395b7b23c37eefe0f426de018123c8794a777174a6a989 WHIRLPOOL 2cfb6f0788833750ceedfcb35b8be37d752392165864ffefeffa5cc04f44a6af1f4cdc758b21044763026765107302cc7966cdf6682e3d49ca1c965a091b569b
 AUX systemd/minecraft.conf 109 SHA256 f667ed5e45b6a68d11904d1fa8d639754ed4e319825be72f536dfd4e8664b0d8 SHA512 d75cf5ee192828c6e253194a163150768361b27c62a561952436c51819ffcf7a30d3402b31ae511d050bf8a6a405585960c04c7de03795673d2384085f0728b4 WHIRLPOOL bb231dce7cf1262de4b73217e54a151bee0eff6c99d626ed2bdcbd1a31e18f3c7213b351979c37ef6ae95eefec743b82a05224f4e854e8e39157c3da248edfa4
-EBUILD minecraft-common-20130203.ebuild 1127 SHA256 420771d311bf34f13365d29a8fdb7f09d1eb60e37bfd42bee2f7f86508b41aad SHA512 6b489c3c675a0a044397bd488d8c04afddd020bf646d1e8239fe3c752ef8cb091cd336dca4c11da195af25ada99b9d8837b581cff99b0b633857c6fac7b537ee WHIRLPOOL f20d9ad16ab2e76a9937881b4c8d44d8cafe3b54cf797824dd2021769cb4ae87e4aaafa562a3642012725df7a3ef7a451eb2f9cb45bdf24151ae3b56bae94fd7
+EBUILD minecraft-common-1.7.ebuild 1127 SHA256 420771d311bf34f13365d29a8fdb7f09d1eb60e37bfd42bee2f7f86508b41aad SHA512 6b489c3c675a0a044397bd488d8c04afddd020bf646d1e8239fe3c752ef8cb091cd336dca4c11da195af25ada99b9d8837b581cff99b0b633857c6fac7b537ee WHIRLPOOL f20d9ad16ab2e76a9937881b4c8d44d8cafe3b54cf797824dd2021769cb4ae87e4aaafa562a3642012725df7a3ef7a451eb2f9cb45bdf24151ae3b56bae94fd7
 MISC ChangeLog 544 SHA256 9059bfdd9f4171df7a3db34b910beb05d7bdd9d0410263ef2326e67eae1ad5c6 SHA512 490e6248d3cd584dd825100a13bcb747da57cf992fa71d6430a407fe314c913dcffca40febc5932a97aed464d2f63867655db35d140aa5f0bf436a0c71eac85e WHIRLPOOL 24e1b8db0f5634657fb6aa0e2fc4ddb610ac5b8283bd3732cb3df23b0863ede7bbaa17ab38ba1336093f443c47ab64c55bc0d6acd448efae87475e9cc3a36ef8
 MISC metadata.xml 481 SHA256 1d3860bb6e1bf8fc2db4b311b0bc10ca4eb747c533836519dffed46edeb1cda3 SHA512 da911fe9c9e1b6f39283a1677a6b2954282a0c65d5e010ac27fd02d7fe5f369abe572282904234794556c3c35ad0bb8ee8602d3876498071b13807b73c112c12 WHIRLPOOL 9b7e66b67b20df72c60aa9af14e6e1237b805aa4107e24d9c880bab06479fd437731f3b42331a2a1e37e7b8d1c2b679f0ffc16cb60a3a8f57ed8736b81bc06d7
diff --git a/games-server/minecraft-common/files/init.sh b/games-server/minecraft-common/files/init.sh
index 4eac995..c0cf128 100644
--- a/games-server/minecraft-common/files/init.sh
+++ b/games-server/minecraft-common/files/init.sh
@@ -8,7 +8,7 @@ extra_started_commands="console"
 MULTIVERSE="${SVCNAME#*.}"
 [[ "${SVCNAME}" == "${MULTIVERSE}" ]] && MULTIVERSE="main"
 
-LOCK="/var/lib/minecraft/${MULTIVERSE}/server.log.lck"
+LOCK="/var/lib/minecraft/${MULTIVERSE}/logs/latest.log"
 PID="/var/run/minecraft/${MULTIVERSE}.pid"
 SOCKET="/tmp/tmux-minecraft-${MULTIVERSE}"
 
@@ -35,7 +35,14 @@ start() {
 	local CMD="umask 027 && '${EXE}' '${MULTIVERSE}'"
 	su -c "/usr/bin/tmux -S '${SOCKET}' new-session -n 'minecraft-${MULTIVERSE}' -d \"${CMD}\"" "@GAMES_USER_DED@"
 
-	if ewaitfile 15 "${LOCK}" && local FUSER=$(fuser "${LOCK}" 2> /dev/null); then
+	local FUSER
+	local SECONDS=15
+	while ! FUSER=$(fuser "${LOCK}" 2> /dev/null) && [ $SECONDS -ne 0 ]; do
+		SECONDS=$(($SECONDS - 1))
+		sleep 1
+	done
+
+	if [ -n "${FUSER}" ]; then
 		echo "${FUSER}" > "${PID}"
 		eend 0
 	else
diff --git a/games-server/minecraft-common/minecraft-common-20130203.ebuild b/games-server/minecraft-common/minecraft-common-1.7.ebuild
similarity index 100%
rename from games-server/minecraft-common/minecraft-common-20130203.ebuild
rename to games-server/minecraft-common/minecraft-common-1.7.ebuild
diff --git a/games-server/minecraft-server/Manifest b/games-server/minecraft-server/Manifest
index 54e2823..5bf576c 100644
--- a/games-server/minecraft-server/Manifest
+++ b/games-server/minecraft-server/Manifest
@@ -1,5 +1,5 @@
 AUX directory.sh 1078 SHA256 935755ee0fdf339cb3c757567e2da278dab8d19a3fbaa1723ced69920b7bed9f SHA512 36030ce7cbe825da7af167605870b7e117fc0f7ca550756a1512899035a422cc7b207baa47bfda6f476050a015b96438e2a24e7bf44b4c58ddd5682f14ada98e WHIRLPOOL 36398cd695abe596c95b29f9fe245e7599db1d71e843dfb837842d6c38a415d6a8977c1bcb6e5fba1901e31ea46ac850e79f3ab026a882e1628b4c625621ca55
-DIST minecraft-server-1.4.7.jar 2061638 SHA256 96b7512aead2fb20ddf780d7dd74208d77f209e16058ea8944150179e65b4dd3 SHA512 ffe75e94ff5383f6cea42c42a5c06bb273d1082c6872f9ba05d2d2e6d4a915f231f8338c5d5c79c9aff70bb69b4d3e4f4eaaf2cabcad0aed2c48713f5dda046c WHIRLPOOL b9f10e48cbf76a6e4c82f698f4f61b9c6274a323df951af73a396623a54e3a1d81351129e9df2ea8b6024b675f0dd67accee7aafb030f4dbe9cb930550762b06
-EBUILD minecraft-server-1.4.7.ebuild 2422 SHA256 24d650b2a9b6ccd03cfb9a44fbe73921f09134cf00e804a15c8e4c046a4e3fb8 SHA512 294559c7e3524eca74e6c252a7eddb3d87648e8e55967d607ea5c07044c7a1af72d2a82d9b156b76adf3bf56a8524193ae3c796da8025b8c732a2d2fcd77d6a5 WHIRLPOOL f51a4e7ca38acbd144d8d2b2fdc922012824382cc761099cd2479a81fa88e659ed766fb7557dbdfc0cb999cca0b43df70510af497140c6e5d22863fe3d0b7caf
+DIST minecraft-server-1.7.4.jar 9170551 SHA256 796d6ca283861a3185f2e87ec321b1233540cdf2638da6e00f1d96d47791031f SHA512 1b4b41a9906a21da1cbf4d96b83c4ebd7b6235c3fc52d42a8957101279ad681f067ef2882059e5e5b3223d2580ff59b60cc575b8ba10f4c5915b1cf5908525c2 WHIRLPOOL 2a99b5c625dac59d89d658ae953c51080f33a15ac89ef865179413641cc2e2fab1b45c3e74d11579e68c96c2db8aa81e178edeee4debd6665b69047eb4549f3c
+EBUILD minecraft-server-1.7.4.ebuild 2453 SHA256 61c310e3fde84889f2768bf648ef6516f49feddd979b2aaa6caa7afd373f5107 SHA512 ae0d21ec2ab3bad6c522b1b0b294fc8ce0b8f5f3270be8940beb3124eabef7a5d8e1502faeeb285f1d0f12aef95fb31c57089ad0db5a0449fe41651e514f439d WHIRLPOOL 4b4666f514fd224aba29a032251a141d03087b0c163b81912687cc3d609bdf32a471d50614b2a73a76861df80c5e687a784387085e8b189c0b13da06f0a79e56
 MISC ChangeLog 7099 SHA256 3b038f9d6c6ce290ddacf0a701e5f6bd961927a25822556312d939a7dc683e74 SHA512 8c149ed3bdb8680c8ca2f0ca585b07253ad20dd809a3cd2866770a2191d603c8ac1a7840fe3c7b6d9457c0f97f17d3908191aec207d70e73a85606fed474c1c6 WHIRLPOOL 30753618e0f8c1e6a436d081e2eb09674677cb4e9c62dc81e33fcd3b0f5f10bc5b4b0aec429ac1a1258482747e3045fcf15418f8ecf9f22697b420927f08f93c
 MISC metadata.xml 468 SHA256 464c232961f7ad78106ac01d7f326f75f7be4eb38779b87164cc7b42c205dd0b SHA512 69cd592b42c8d027360508eae7238f90343417ab5ac245120765e6c69ceeb779cebb7e14989d1f58f85e44e85c1cf0b73abecd54fd294a21cca2b350913397a5 WHIRLPOOL f460928cd6507d639ba3636c695ae063a3c1785abd8a23b7b7ba314de90d91391f55ee146f641fc9223bf905ef7cb6f1a8fe04791746bc3b8624b9061a2f69c7
diff --git a/games-server/minecraft-server/minecraft-server-1.4.7.ebuild b/games-server/minecraft-server/minecraft-server-1.7.4.ebuild
similarity index 94%
rename from games-server/minecraft-server/minecraft-server-1.4.7.ebuild
rename to games-server/minecraft-server/minecraft-server-1.7.4.ebuild
index eb8ca5e..40a66a2 100644
--- a/games-server/minecraft-server/minecraft-server-1.4.7.ebuild
+++ b/games-server/minecraft-server/minecraft-server-1.7.4.ebuild
@@ -7,7 +7,7 @@ inherit games java-pkg-2
 
 DESCRIPTION="Official dedicated server for Minecraft"
 HOMEPAGE="http://www.minecraft.net"
-SRC_URI="http://assets.minecraft.net/${PV//./_}/minecraft_server.jar -> ${P}.jar"
+SRC_URI="https://s3.amazonaws.com/Minecraft.Download/versions/${PV}/minecraft_server.${PV}.jar -> ${P}.jar"
 LICENSE="as-is"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
@@ -16,7 +16,7 @@ RESTRICT="mirror"
 
 DEPEND=""
 RDEPEND=">=virtual/jre-1.6
-	games-server/minecraft-common"
+	~games-server/minecraft-common-1.7"
 
 S="${WORKDIR}"
 
-- 
1.8.3.2

